snakemake/persistence.py

```
@@ -12,7 +12,7 @@
 import tempfile
 import time
 from base64 import urlsafe_b64encode, b64encode
-from functools import lru_cache
+from functools import lru_cache, partial
```

Ruff

`functools.partial` imported but unused

Remove unused import: `functools.partial`

(F401)